examples/StandAloneMatLab/testPy.m

```diff
@@ -0,0 +1,4 @@
+function out = testPy(z)

```

Comment:
Documentation update with reference to these sample matlab files would certainly help